src/app/pages/courses-page/courses-page.component.spec.ts

```diff
@@ -1,25 +1,82 @@
+import { Component, Input } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+
+import { Course } from '../../interfaces/course.interface';
+import { CoursesService } from '../../services/courses.service';
+import { MockCoursesService } from '../../services/courses.service.mock';
 import { CoursesPageComponent } from './courses-page.component';
 
+@Component({ selector: 'app-search-input' })
```

Comment:
keep in mind that you will need to add/update inputs/outputs when your components change

Reply:
Okay